nano/test_common/system.cpp

```diff
@@ -184,9 +207,28 @@ std::shared_ptr<nano::node> nano::test::system::make_disconnected_node (std::opt
 		return nullptr;
 	}
 	node->start ();
+	nodes.push_back (node);
```

Comment:
A disconnected node cannot be added to the list of system nodes because then it will not remain disconnected. Any nodes added after the disconnected node will connect to the disconnected node.

We could create a second set nodes to hold the disconnected nodes.

Reply:
Good spot. Fixed it.